crates/data-model/Cargo.toml

```diff
@@ -22,7 +22,9 @@ rand.workspace = true
 rand_chacha = "0.3.1"
 regex = "1.11.0"
 woothee = "0.13.0"
+ruma-common = "0.13.0"
 
 mas-iana.workspace = true
 mas-jose.workspace = true
+mas-matrix.workspace = true
```

Comment:
I would like to avoid this mas-data-model -> mas-matrix dependency. Can we instead pass the homeserver name as a string instead of the homeserver connection?